src/Components/Search/SearchBarPills.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  CarouselNext,
+  DROP_SHADOW,
+  Flex,
+  Pill,
+  ShelfNavigationProps,
+} from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import { Carousel, CarouselPrevious } from "Components/Carousel"
+import styled, { css } from "styled-components"
+
+const SearchPills = [
+  "TOP",
+  "ARTWORK",
+  "ARTIST",
+  "ARTICLE",
+  "SALE",
+  "ARTIST_SERIES",
+  "COLLECTION",
+  "FAIR",
+  "SHOW",
+  "GALLERY",
+]
+
+const STATES = {
+  hover: css`
+    outline: 0;
+    color: ${themeGet("colors.black100")};
+    box-shadow: ${DROP_SHADOW};
+  `,
+  focus: css`
+    outline: 0;
+    color: ${themeGet("colors.black100")};
+    border-color: ${themeGet("colors.brand")};
+  `,
+  disabled: css`
+    opacity: 0;
+    cursor: default;
+  `,
+}
+
+const ChevronStyle = css`
+  transform: "translateX(0)";
+  background-color: "white";
+  color: ${themeGet("colors.black60")};
+  height: 30px;
+  width: 30px;
+  border: 1px solid ${themeGet("colors.black5")};
+  border-radius: 50%;
+  pointer-events: auto;
+
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  user-select: none;
+  transition: opacity 250ms, color 250ms, border-color 250ms,
+    box-shadow 0.25s ease;
+
+  ${props => {
+    return css`
+    ${props.hover && STATES.hover}
+    ${props.focus && STATES.focus}
+    ${props.disabled && STATES.focus}
+  `
+  }}
+
+  &:hover {
+    ${STATES.hover}
+  }
+  &:focus {
+    ${STATES.focus}
+  }
+
+  &:disabled {
+    ${STATES.disabled}
+  }
+
+  @media (hover: none) {
+    display: none;
+  }
+`
+
+export const SearchBarPills = () => {
+  return (
+    <Flex p={2}>
+      <Carousel
```

Comment:
Faced with the problem that `Carousel` cannot be scrolled with the mouse /touchpad. Therefore, it may be better to use `Flex` with `overflow-x: scroll` and without chevron arrows. Please remind me to discuss this moment with you tomorrow 🙏

Reply:
or use the `Shelf` component, but adding the required changes ([the possibility of pagination by cell](https://github.com/artsy/palette/tree/dimatretyak/sandbox/shelf), it is possible to pass custom navigation buttons, etc)